crates/sb_module_loader/standalone/mod.rs

```diff
@@ -87,7 +87,12 @@ where
 
     // use a dummy npm registry url
     let npm_registry_url = ModuleSpecifier::parse("https://localhost/").unwrap();
-    let root_path = std::env::temp_dir().join(format!("sb-compile-{}", current_exe_name));
+    let root_path = if cfg!(target_family = "unix") {
+        PathBuf::from("/var/tmp")
```

Comment:
why does it need to be `/var/tmp` instead of `/tmp`?

Reply:
This was changed because if the contents of eszip binary is mounted under `/tmp`, it will conflict with tmpfs and become inaccessible.

Reply:
ah gotcha!